cmd/utils/flags.go

```diff
@@ -847,6 +848,12 @@ var (
 		Usage: "Enable circuit capacity check during block validation",
 	}
 
+	CircuitCapacityCheckWorkersFlag = cli.UintFlag{
```

Comment:
Fyi you have to add this in another file (forgot which) if you want it to be printed with `geth --help`.

Reply:
thanks, done in https://github.com/scroll-tech/go-ethereum/pull/996/commits/3e692e4cd03d6972501fe42b001c25d8107a45f7